### PR TITLE
Make planar wave structure for acoustic expl

### PR DESCRIPTION
**Pull request by krenzland**
Note: This leads to a small difference in the error (and is conceptually cleaner)

### CODE REVIEW

File: src/Solver/Interoperability.cpp [hunk 1]
```diff
@@ -987,12 +987,43 @@ void seissol::Interoperability::initInitialConditions()
...
+      varField = {1, 8};
+      ampField = {1.0, 1.0};
+    }
+#else
+    //Poroelastic materials have the following wave modes:
+    //-P, -S2, -S1, -Ps, N, N, N, N, N, Ps, S1, S2, P
+    //Here we impose -S1, -Ps and P
+    std::vector<int> varField = {2,3,12};
+    std::vector<std::complex<double>> ampField = {1.0, 1.0, 1.0};
+#endif
```
Comment (sebwolf-de) [inline src/Solver/Interoperability.cpp:1014]: Why did you move this initializer here? It is already implemented here (without the check for mu): https://github.com/SeisSol/SeisSol/blob/master/src/Physics/InitialField.h#L39-L51
IMHO this change spaghettifies the code.
I'd prefer, if you add you changes regard `mu` in `InitialField.h`.

**Review by sebwolf-de — CHANGES_REQUESTED** [review]

File: src/Solver/Interoperability.cpp (hunk 1 above)
Comment (krenzland) [reply src/Solver/Interoperability.cpp:1014]: Because this doesn't work like you're suggesting. It doesn't make sense to pass the variable field explicitly and then change the fields for the acoustic wave equation implicitly. Maybe it'd make sense to add methods that return default parameters?

Comment (sebwolf-de) [reply src/Solver/Interoperability.cpp:1014]: Yep, like a PlanarWaveFactory ;-)

**Review by sebwolf-de — APPROVED** [review]

**Comment by sebwolf-de** [comment]
`double muBar = (c44 + c55 + c66) / 3.0;` ;-)
